### PR TITLE
chore: Leverage isEmpty vs size equality zero checks

### PR DESCRIPTION
**Pull request by kingthorin**
- Use isEmpty() instead, as pointed out by SAST (SonarLint)
- Skipped occurrence in importLogFiles as it's going to be removed.
- Updated CHANGELOGs with maintenance note where add-ons didn't already have one.
- BugTrackerBugZillaParam > Remove blocks of commented code.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
This repo only has one occurrence of `size() != 0` which is in the dirbuster code.

**Comment by thc202** [comment]
Worth changing anyway, we already did more fundamental changes.

File: addOns/bugtracker/src/main/java/org/zaproxy/zap/extension/bugtracker/BugTrackerBugzillaParam.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ protected void parse() {
             // this.enabledConfigsNames = new ArrayList<>(DEFAULT_CONFIGS_NAMES.length);
         }
 
-        // if (this.configs.size() == 0) {
+        // if (this.configs.isEmpty()) {
```
Comment (thc202) [inline addOns/bugtracker/src/main/java/org/zaproxy/zap/extension/bugtracker/BugTrackerBugzillaParam.java:81]: Or remove the (all) commented code?

**Comment by kingthorin** [comment]
Addressed.

**Comment by thc202** [comment]
I was suggesting remove all of the commented code in the class (i.e. usages of `enabledConfigsNames`).

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**